dbgap_auto_tool/graph.py

```diff
@@ -0,0 +1,224 @@
+import json
```

Comment:
For graph and node.py I think you should use `data-simulator` as dependency so that we don't have maintains the code 2 places. This version does not support `array` type I think